src/Components/Components/src/Json/JsonUtil.cs

```diff
@@ -1,34 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.JSInterop;
-using System;
-
-namespace Microsoft.AspNetCore.Components
-{
-    /// <summary>
-    /// Provides mechanisms for converting between .NET objects and JSON strings.
-    /// </summary>
-    [Obsolete("Use Microsoft.JSInterop.Json instead.")]
-    public static class JsonUtil
```

Comment:
/cc @SteveSandersonMS

Reply:
Thanks for pointing it out. This removal is fine with me. It has to happen sooner or later, and this has been marked `[Obsolete]` for at least a couple of Blazor releases by now.